include/iso15118/d20/session.hpp

```diff
@@ -8,69 +8,71 @@
 #include <variant>
 #include <vector>
 
-#include <iso15118/message/common.hpp>
+#include <iso15118/message/msg_data_types.hpp>
 
 namespace iso15118::d20 {
 
+namespace datatypes = message_20::datatypes;
```

Comment:
Thanks for the tip. This affects quite a few files, especially the state machine cpp files.
I'll see how I can best solve this

Reply:
It's correct, that one should be careful with namespace aliases, that become visible everywhere.  From time to time it might be still useful.  Just another thing: we might also use `dt` as a short abbreviation for `datatypes`

Reply:
`dt` is a good idea. `iso15118::message_20::datatypes::` is quite long.

Reply:
I added `dt` as namespace alias for some src files.